README.md

```diff
@@ -315,20 +350,6 @@ if Rails.configuration.unleash.is_enabled? "AwesomeFeature", @unleash_context
 end
 ```
 
-If you don't want to check a feature is disabled with `unless`, you can also use `is_disabled?`:
```

Comment:
Why is this example removed?

Reply:
I'm with Gard, I'd like this to stay, it's part of the public API and needs to be surfaced here

Reply:
There was no other mention of using `unless` anywhere in the document, so it felt weird to say "if you don't want to use this thing I never told you about"

Reply:
unless is built in to ruby, the example is for `is_disabled?` usage